src/Kdyby/Doctrine/Entities/BaseEntity.php

```diff
@@ -25,6 +25,7 @@
 /**
  * @author Filip Procházka <filip@prochazka.su>
  *
+ * @deprecated
  * @ORM\MappedSuperclass()
  */
 abstract class BaseEntity extends Nette\Object implements \Serializable
```

Comment:
Also not sure which min PHP is code targeted for. Composer is missing it, here is used PHP 5.3 array syntax and Readme has PHP 5.4.
Can send PR if you pick some.